src/gallery/slider.js

```diff
@@ -260,6 +315,30 @@ const clientWidth = window.innerWidth,
 		} )
 	},
 
+	toggleFocusMode = () => {
+		const toggleElements = [
```

Comment:
Did you consider only flipping a class here and apply the proper show/hide in CSS?

Reply:
Seems like a good idea, let me look into that

Reply:
How about https://github.com/wikimedia/wikipedia-preview/pull/86/commits/ba7d7f7de3895487f2c25148ea003899136a4c5b?

Reply:
I think the class should be put higher up on a single element. Which elements should be shown or hidden can be handled in CSS.

Reply:
Ah I think I see what you meant with your first comment, check out https://github.com/wikimedia/wikipedia-preview/pull/86/commits/3248fae8e6af0117bb5d8324d28f268b1412390a. I like this _much_ better, fully takes advantage of the cascading property of CSS to simplify and: 

* get rid of the nested looping,
* get rid of `focusMode` state variable
* make it more readable and maintainable

Thanks for the great feedback!